conf/default.yaml

```diff
@@ -80,6 +80,8 @@ logs:
 #  token: false
 #  # support for the new v1 search endpoint, functional by incomplete read more on ticket 1732
 #  search: false
+#  # disable writing body size to logs, read more on ticket 2013
+#  bytesin_off: false
```

Comment:
I think #2012 describes better than #2013

Reply:
What do you mean?

Reply:
upss sorry I meant here https://github.com/verdaccio/verdaccio/issues/1912#issuecomment-678674600 ticket 1912

Reply:
Fixed
Amend + force push or new commit? What do you prefer?